src/commands/serve.rs

```diff
@@ -57,6 +58,19 @@ pub(crate) fn serve(args: &ArgMatches) {
     }
 }
 
+fn parse_default_port(port: &str) -> u16 {
```

Comment:
`get_port_or_fallback` ? making up good names can be so hard ...

Reply:
I'd say `get_available_http_port`, WDYT?